dwave/system/samplers/clique.py

```diff
@@ -165,17 +165,16 @@ def sample(self, bqm, chain_strength=None, **kwargs):
 
         if chain_strength is None:
             # chain length determines chain strength
-            if embedding:
-                chain_strength = 1.5 * math.sqrt(len(embedding))
+            if embedding and bqm.num_variables > 1:
+                squared_j = (j ** 2 for j in bqm.quadratic.values())
+                num_edges = (bqm.num_variables*(bqm.num_variables-1))/2
+                rms = math.sqrt(sum(squared_j)/num_edges)
+       
+                chain_strength = 1.5 * rms * math.sqrt(bqm.num_variables)
```

Comment:
I think here we should do a quick fix: if one passes a fully connected bqm then `math.sqrt(bqm.num_variables)` is correct. In cases someone passes a sparser BQM, then number of variables should be replaced with average degree

Reply:
Is that true even though we're always using a clique embedding?

Reply:
in all my previous tests, that formula seems quite general when we use the average degree, in terms of the clique.